course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -255,6 +342,28 @@
         $scope.renderName = function(data, type, full, meta) {
             return full.profile.name_last + ', ' + full.profile.name_first;
         };
+        $scope.renderRemove = function(data, type, full, meta) {
+            // TODO - maybe make this a directive?  the isolate scope
+            //        in a directive complicates things.  anything has
+            //        to be better than this mess, though.
```

Comment:
Although it would require more actual concatenations, it might read easier if it was built in a nested way, not a linear way. Not sure if that would work for all cases, though.

e.g.:
```
cellStyle = registrarFed ? ' style="color: #ccc;"' : '';
cellLinkOpening = !registrarFed ? .... etc
...
cell = '<i class="fa fa-trash-o"' + cellStyle + '></i>';
cell = cellLinkOpening + cell + cellLinkClosing;
cell = '<div ...' + cell + '</div>';
return cell;
```

Also since we're using `registrarFed` logic in multiple places would it make sense to make it a computed property of the row data and refer to that instead of re-running the regex each time? Won't save much time with only 10 records being displayed, I suppose. But we have sorting on the Manually/Registrar added field, so seems like we should be able to just reference that somehow.